### PR TITLE
Removes old Monocular API deployment from the Helm chart

### PR DESCRIPTION
**Pull request by prydonius**
Not bumping chart version as it will be bumped when we're ready to
release 1.0.

Signed-off-by: Adnan Abdulhussein <adnan@bitnami.com>

### CODE REVIEW

**Comment by migmartri** [comment]
It might make sense to attach the diff of the rendered version of the chart?

File: deployment/monocular/templates/ui-deployment.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ spec:
       labels:
         app: {{ template "fullname" . }}-ui
       annotations:
-        checksum/config: {{ print .Values.api.config | sha256sum }}
+        checksum/config: {{ include (print $.Template.BasePath "/ui-config.yaml") . | sha256sum }}
```
Comment (migmartri) [inline deployment/monocular/templates/ui-deployment.yaml]: What's the difference here? That you are actually now rendering the content?

File: deployment/monocular/values.yaml [hunk 2]
```diff
@@ -59,63 +59,6 @@ chartsvc:
   tolerations: []
   affinity: {}
 
-# OLD API CONFIG
-api:
```
Comment (migmartri) [inline deployment/monocular/values.yaml]: Nice! :boom:

File: deployment/monocular/templates/ui-deployment.yaml (hunk 1 above)
Comment (prydonius) [reply deployment/monocular/templates/ui-deployment.yaml]: Yes, honestly I'm not sure why it was this way before, the Helm documentation recommends doing it this way: https://github.com/helm/helm/blob/master/docs/charts_tips_and_tricks.md#automatically-roll-deployments-when-configmaps-or-secrets-change

**Comment by prydonius** [comment]
Diff:

```
--- oldtemplate.txt	2018-09-20 16:15:49.000000000 -0700
+++ newtemplate.txt	2018-09-20 16:15:34.000000000 -0700
@@ -11,42 +11,9 @@
     heritage: "Tiller"
 type: Opaque
 data:
-  mongodb-root-password: "OUc0WnlxUURQSg=="
+  mongodb-root-password: "VGg1a0NKSldOeA=="
 
 ---
-# Source: monocular/templates/api-config.yaml
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: RELEASE-NAME-monocular-api-config
-  labels:
-    app: RELEASE-NAME-monocular
-    chart: "monocular-0.6.4"
-    release: "RELEASE-NAME"
-    heritage: "Tiller"
-data:
-  monocular.yaml: |-
-    mongodb:
-      url: RELEASE-NAME-mongodb:27017
-      database: 
-    cacheRefreshInterval: 3600
-    cors:
-      allowed_headers:
-      - content-type
-      - x-xsrf-token
-      allowed_origins:
-      - ""
-    releasesEnabled: true
-    repos:
-    - name: stable
-      source: https://github.com/kubernetes/charts/tree/master/stable
-      url: https://kubernetes-charts.storage.googleapis.com
-    - name: incubator
-      source: https://github.com/kubernetes/charts/tree/master/incubator
-      url: https://kubernetes-charts-incubator.storage.googleapis.com
-    tillerNamespace: kube-system
-    
----
 # Source: monocular/templates/ui-config.yaml
 apiVersion: v1
 kind: ConfigMap
@@ -64,7 +31,6 @@
         googleAnalyticsId: 'UA-XXXXXX-X',
         appName: 'Monocular',
         backendHostname: '/api',
-        releasesEnabled: true
       }
     };
 
@@ -155,24 +121,6 @@
     release: "RELEASE-NAME"
 
 ---
-# Source: monocular/templates/api-service.yaml
-apiVersion: v1
-kind: Service
-metadata:
-  name: RELEASE-NAME-monocular-api
-  labels:
-    chart: "monocular-0.6.4"
-spec:
-  type: NodePort
-  ports:
-  - port: 80
-    targetPort: 8081
-    protocol: TCP
-    name: monocular-api
-  selector:
-    app: RELEASE-NAME-monocular-api
-
----
 # Source: monocular/templates/chartsvc-service.yaml
 apiVersion: v1
 kind: Service
@@ -304,64 +252,6 @@
       - name: data
         emptyDir: {}
 ---
-# Source: monocular/templates/api-deployment.yaml
-apiVersion: extensions/v1beta1
-kind: Deployment
-metadata:
-  name: RELEASE-NAME-monocular-api
-  labels:
-    chart: "monocular-0.6.4"
-spec:
-  replicas: 2
-  template:
-    metadata:
-      labels:
-        app: RELEASE-NAME-monocular-api
-      annotations:
-        checksum/config: adad6036a6ed7da9dfd3d60adf57208a0a31357733c9bb2a59b2d331f86be969
-    spec:
-      containers:
-      - name: monocular
-        image: "bitnami/monocular-api:v0.7.3"
-        imagePullPolicy: Always
-        env:
-        - name: MONOCULAR_HOME
-          value: /monocular
-        ports:
-        - containerPort: 8081
-        livenessProbe:
-          httpGet:
-            path: /healthz
-            port: 8081
-          initialDelaySeconds: 180
-          timeoutSeconds: 10
-        readinessProbe:
-          httpGet:
-            path: /healthz
-            port: 8081
-          initialDelaySeconds: 30
-          timeoutSeconds: 5
-        volumeMounts:
-          - name: cache
-            mountPath: /monocular
-          - name: config
-            mountPath: /monocular/config
-        resources:
-            limits:
-              cpu: 200m
-              memory: 512Mi
-            requests:
-              cpu: 100m
-              memory: 256Mi
-            
-      volumes:
-      - name: config
-        configMap:
-          name: RELEASE-NAME-monocular-api-config
-      - name: cache
-        emptyDir: {}
-
----
 # Source: monocular/templates/chartsvc-deployment.yaml
 apiVersion: apps/v1beta2
 kind: Deployment
@@ -464,7 +354,7 @@
       labels:
         app: RELEASE-NAME-monocular-ui
       annotations:
-        checksum/config: 692b42632aa64d809d310293ff6cd6f391a2db7adbbfffca55fa62e701603e39
+        checksum/config: 6405b24d53ef21d0f2195a176cc160f35d51bbee7f0440d3cb77c6f4c2fe48c6
     spec:
       containers:
       - name: monocular
@@ -511,7 +401,7 @@
 apiVersion: batch/v1
 kind: Job
 metadata:
-  name: RELEASE-NAME-monocular-sync-initial-stable-cv4tp
+  name: RELEASE-NAME-monocular-sync-initial-stable-u3wvo
   labels:
     monocular.helm.sh/repo-name: stable
     app: RELEASE-NAME-monocular
@@ -551,7 +441,7 @@
 apiVersion: batch/v1
 kind: Job
 metadata:
-  name: RELEASE-NAME-monocular-sync-initial-incubator-m9chi
+  name: RELEASE-NAME-monocular-sync-initial-incubator-3oeey
   labels:
     monocular.helm.sh/repo-name: incubator
     app: RELEASE-NAME-monocular
@@ -712,10 +602,6 @@
           servicePort: 80
         path: /
       - backend:
-          serviceName: RELEASE-NAME-monocular-api
-          servicePort: 80
-        path: /api/
-      - backend:
           serviceName: RELEASE-NAME-monocular-chartsvc
           servicePort: 8080
         path: /api/chartsvc

```

**Pull request merged**